OpenRA.Game/Map/MapPreview.cs

```diff
@@ -116,7 +116,7 @@ public void UpdateFromMap(Map m, MapClassification classification)
 			Type = m.Type;
 			Type = m.Type;
 			Author = m.Author;
-			PlayerCount = m.Players.Count(x => x.Value.Playable);
+			PlayerCount = m.PlayerCount;
```

Comment:
It might be cleaner to move the yaml querying here instead?  Everything in the UI code that interacts with a map goes via the MapPreview anyway (because these support remote maps as well as local), and it would be nice to cut down some of the random functions off Map.